examples/nlp/language_modeling/tuning/conf/megatron_gpt_sft.yaml

```diff
@@ -109,15 +110,15 @@ model:
       names: null # Names of the corresponding datasets used to log metrics.
       global_batch_size: ${model.global_batch_size}
       micro_batch_size: ${model.micro_batch_size}
-      shuffle: True
+      shuffle: False
       num_workers: 4
       memmap_workers: ${model.data.train_ds.memmap_workers}
       pin_memory: True
       max_seq_length: 2048
       min_seq_length: 1
-      drop_last: True
-      context_key: 'input'
-      label_key: 'output'
+      drop_last: False
```

Comment:
Why we drop last if we evaluate with the validation dataset?

Reply:
I have one use case that I have to drop the last because I am preparing a dataset that computes contrastive loss.  I need to make sure all the batch has the same batch size.

Reply:
If we set `drop_last=False`, we can use `pad_samples_to_global_batch_size=True` right? Or this doesn't fulfill your settings? https://github.com/NVIDIA/NeMo/blob/fix-gpt-trainer-test/nemo/collections/nlp/models/language_modeling/megatron_gpt_sft_model.py#L788 
If we set default `drop_last=True`, then we may drop some samples for evaluation which may show incorrect results for comparison.